nbconvert/templates/html/basic.tpl

```diff
@@ -209,3 +209,27 @@ var element = $('#{{ div_id }}');
 </script>
 </div>
 {%- endblock -%}
+
+{%- block data_widget_state scoped %}
+{% set div_id = uuid4() %}
+{% set datatype = output.data|filter_data_type %} 
+<div id="{{ div_id }}"></div>
+<div class="output_subarea output_widget_state {{extra_class}}">
+<script type="text/javascript">
+var element = $('#{{ div_id }}');
+</script>
+<script type="{{datatype}}">{{output.data[datatype]}}</script>
+</div>
+{%- endblock data_widget_state -%}
+
+{%- block data_widget_view scoped %}
+{% set div_id = uuid4() %}
+{% set datatype = output.data|filter_data_type %} 
+<div id="{{ div_id }}"></div>
+<div class="output_subarea output_widget_view {{extra_class}}">
+<script type="text/javascript">
+var element = $('#{{ div_id }}');
+</script>
+<script type="{{datatype}}">{{output.data[datatype]}}</script>
```

Comment:
I think that this need to be json-serialized, see the change in https://github.com/spatialaudio/nbsphinx/pull/76